src/modules/doctors/entities/doctor.entity.ts

```diff
@@ -35,6 +35,9 @@ export class Doctor {
   @OneToMany(() => Room, (room) => room.doctor)
   rooms: Room;
 
+  @Column('uuid', { array: true, nullable: true })
+  associatedRooms: string[];
```

Comment:
Don't you have already a relation to the room? What is the purpose of this?

Reply:
https://futurotech.atlassian.net/jira/software/projects/FT/boards/50?selectedIssue=FT-64&text=64

> If a room has a doctor associated, the association should be removed

Maybe I misunderstood. I created an explicit relation.

Reply:
I think you don't need this column, your association with the room is handled by the doctorId field in the room entity, so when you remove this entity association will be removed implicitly.